packages/server/src/bandit/banditSelection.ts

```diff
@@ -46,6 +46,15 @@ function selectRandomVariant(test: EpicTest): Result {
     };
 }
 
+export function epsilonValueForBanditTest(testBanditData: BanditData): number {
+    if (testBanditData.testName.includes('2024-07-20_BANDIT_EPSILON1_VARIANTS')) {
+        return 1;
+    } else if (testBanditData.testName.includes('2024-07-20_BANDIT_EPSILON2_VARIANTS')) {
+        return 0.5;
+    }
+    return 0;
```

Comment:
not sure what `epsilon=0` means, always pick the best variant?

Reply:
Oh yes, so if epsilon=0 there will be no random selection at all, so it should be epsilon=1 instead